### PR TITLE
Fix for "product/tag" router

### PR DESCRIPTION
**Pull request by uniquarkD**
Updated the subscription for "product/tag" router.

### CODE REVIEW

File: packages/reaction-core/common/router.js [hunk 1]
```diff
@@ -230,10 +230,8 @@ Router.map(function () {
     controller: ShopController,
     path: "product/tag/:_id",
     template: "products",
-    waitOn: function () {
-      return this.subscribe("Products", Session.get("productScrollLimit"));
-    },
     subscriptions: function () {
+      this.tagSubsription = this.subscribe("Products", Session.get("productScrollLimit"));
```
Comment (aaronjudd) [inline packages/reaction-core/common/router.js]: `this.tagSubscription` (spelling)

Comment (uniquarkD) [inline packages/reaction-core/common/router.js]: Oh, that's embarrassing.

**Comment by aaronjudd** [comment]
@uniquarkD Can you post a short description about what this fixes? I just want to make sure we know what we're testing for before merging (will merge to a build tomorrow, if we can test).

**Comment by uniquarkD** [comment]
Its infinite scrolling is reloading the page everytime we scroll down the page because we are returning its subscription on "waitOn" which triggers the global loading hook.

**Comment by zenweasel** [comment]
@uniquarkD Hey, these changes are going to be irrelevant in the next release so we are closing pending v12.

**Pull request closed without merging**